### ISSUE TITLE
<select/> popup location is incorrect when enable device emulation

### ISSUE THREAD

**Issue by f111fei** [electron/electron#8486]
* Electron version: v1.4.15
* Operating system: windows7

### Expected behavior

Run in electron v1.3.9

![ok](https://cloud.githubusercontent.com/assets/7069719/22236870/585afdd2-e243-11e6-8c4a-1c87462b3ead.gif)

### Actual behavior

Run in electron v1.4.x

![error](https://cloud.githubusercontent.com/assets/7069719/22236880/6089a346-e243-11e6-9ff0-7a3d1e905cba.gif)

### How to reproduce

Here is my `index.html`
```

<html>

<head>
	<title>Electron Demo</title>
	<style>
		body {
			display: block;
			margin: 0;
		}

		.dropdown {
			position: absolute;
			left: 10px;
			height: 30px;
			line-height: 30px;
			width: 500px;
		}

	</style>
	<script>
		var electron = require('electron');
		var contents = electron.remote.getCurrentWebContents();
		contents.enableDeviceEmulation({
			screenPosition: 'mobile'
		});
	</script>
</head>

<body>
	<select class="dropdown">
		<option>1</option>
		<option>2</option>
		<option>3</option>
		<option>4</option>
		<option>5</option>
	</select>
</body>

</html>

```

**Comment by deepak1556** [comment]
Can reproduce on 1.4.x but it seems to be fixed with 1.5.0, can you try the beta release ?

**Comment by f111fei** [comment]
PC is not by my side, I'll try this after the holidays.  Happy New Year.

**Comment by f111fei** [comment]
I have verified it fixed with 1.5.0.
